homeassistant/components/bmw_connected_drive/lock.py

```diff
@@ -4,10 +4,17 @@
 from bimmer_connected.state import LockState
 
 from homeassistant.components.lock import LockDevice
-from homeassistant.const import STATE_LOCKED, STATE_UNLOCKED
+from homeassistant.const import (
+    ATTR_ATTRIBUTION,
+    STATE_LOCKED,
+    STATE_UNKNOWN,
+    STATE_UNLOCKED,
+)
 
 from . import DOMAIN as BMW_DOMAIN
 
+ATTRIBUTION = "Data provided by BMW Connected Drive"
```

Comment:
Please put this in `bmw_connected_drive/const.py`

Reply:
Done.